### PR TITLE
Package ingest-user-agent as a module

### PR DESCRIPTION
**Pull request by jasontedor**
This commit moves ingest-user-agent from being a plugin to being a module that is packaged with Elasticsearch distributions.

### CODE REVIEW

**Review by martijnvg — APPROVED** [review]
LGTM

**Comment by jasontedor** [comment]
@elasticmachine run gradle build tests 1

**Review by rjernst — APPROVED** [review]
Lgtm

File: distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java [hunk 1]
```diff
@@ -112,11 +112,14 @@ void execute(Terminal terminal, Environment env, String pluginName, boolean purg
         if ((!Files.exists(pluginDir) && !Files.exists(pluginConfigDir) && !Files.exists(removing))
                 || (!Files.exists(pluginDir) && Files.exists(pluginConfigDir) && !purge)) {
 
-            // special case for ingest-geoip since it is a module now but could have been installed from a previous when it was a plugin
-            if ("ingest-geoip".equals(pluginName)) {
+            /*
+             * This is special case handling for ingest-geoip and ingest-user-agent since they are modules now but could have been installed
+             * from a previous when it was a plugin.
```
[the remark this answers is not in the document]
Comment (jasontedor) [reply distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java]: I'll fix it in post.

**Comment by jasontedor** [comment]
Thanks @martijnvg and @rjernst.

**Pull request merged**